### DISCUSSION TITLE
Slice image into separate files

### DISCUSSION THREAD

**Discussion by ned-code** [ImageMagick/ImageMagick#2474] [Help]
Hello,

I'm trying to find a solution to a problem and would appreciate any suggestions.

Our school uses online quizzes but students must submit their responses on paper. The responses are scanned and uploaded to a folder on our server (i.e. Folder A, in the diagram).  Teachers then download the files and match them up with the quiz for grading. This is huge bottleneck for us. 

Our coder can create a plugin that will automatically match the students' responses with the appropriate quiz question IF the filename contains certain identifiers. You can get a better idea of the proposed process here: https://prnt.sc/u70mbs. 

Would ImageMagick be able to do that sort of thing? Any suggestions would be appreciated.

Thanks,
Fern

**Comment by fmw42** [comment]
You will have to post a quiz for us to see what might be involved in separating questions. What is the platform/OS that would be used to do the processing? Do you have ImageMagick? If so, what is the version?
